### ISSUE TITLE
LinkRefiner clarification ?

### ISSUE THREAD

**Issue by lamhoangtung** [clovaai/CRAFT-pytorch#56]
Hi @YoungminBaek. Thanks for the excellent works.

I'm trying to replicate LinkRefiner but not achieve a good result as you did.

May I ask:
1. You said that the width of each line in LinkRefiner ground truth is proportional to the distance between paired control points. How much is it exactly? 40% of the distance between paired control points, 60%, ..etc ?
![image](https://user-images.githubusercontent.com/26834698/66690490-802cc280-ecba-11e9-81e9-605e82f70ece.png)

2. What loss function do you used to train the LinkRefiner? Currently I'm using Focal Loss ;_;

Hope you can answer these question :P

**Comment by YoungminBaek** [comment]
@lamhoangtung Sorry for the late reply.

1. The width of center line was 1/8 of the distance between paired control points.

2. The loss function is the same as the affinity loss. We used MSE loss for link refine too.

Hope this will help. :)

**Comment by lamhoangtung** [comment]
Thanks a lot @YoungminBaek

**Comment by bharatsubedi** [comment]
@lamhoangtung could you please share your link refiner training code and data preparation technique?

**Comment by lamhoangtung** [comment]
@bharatsubedi Unfortunately, I had no intention of releasing it publicly

**Comment by zhangsiqiGit** [comment]
> 

hi，I'm interested and looking for fine-tuning linkrefiner submodule. May I ask your training strategy and hyper params?
Thanks a lot!

**Comment by zhangsiqiGit** [comment]
hi，I'm interested and looking for fine-tuning linkrefiner submodule. May I ask your training strategy and hyper params?
Thanks a lot!

> @bharatsubedi Unfortunately, I had no intention of releasing it publicly

hi，I'm interested and looking for fine-tuning linkrefiner submodule. May I ask your training strategy and hyper params?
Thanks a lot!
